services/web/server/src/simcore_service_webserver/projects/db.py

```diff
@@ -368,6 +370,10 @@ async def list_projects(  # pylint: disable=too-many-arguments
                     projects.join(projects_to_products, isouter=True)
                     .join(access_rights_subquery, isouter=True)
                     .join(folders_to_projects, isouter=True)
+                    if settings.WEBSERVER_FOLDERS
```

Comment:
THOUGHT: i wonder if we could do a little less one-liners with the sql queries. A some point i feel they are even more difficult to read, specially the conditional ones like thisone

Reply:
improved :+1: